myapp/templates/index.html

```diff
@@ -14,6 +14,9 @@
             <th>Name</th>
             <th>Price</th>
             <th>Change</th>
+            {% if request.user.is_authenticated %}

```

Comment:
this can be shortened to this
```suggestion
            {% if user.is_authenticated %}

```

Reply:
I am now using `{% if profile %}` on all templates (except base.html) for consistency